files/ja/web/css/blend-mode/index.html

```diff
@@ -4,33 +4,32 @@
 tags:
   - Blend modes
   - CSS
-  - CSS Compositing
   - CSS Data Type
-  - CSS データ型
-  - CSS 合成
   - Compositing
+  - Compositing and Blending
+  - Data Type
   - Reference
+  - color
```

Comment:
As far as I know, and I'm reasonably confident, the tags never matter in translated content. 
All the KS macros I know of use the en-US document's tags for figuring out what and which links to include.

Reply:
> As far as I know, and I'm reasonably confident, the tags never matter in translated content.
> All the KS macros I know of use the en-US document's tags for figuring out what and which links to include.

So, how should we do? Is it incorrect to adjust tags in translated articles?

Reply:
Yes it is. Because it won't be of any use. You could but all or anything in there and it wouldn't affect anything. 
We currently don't have the ability to filter-search by tags in the new site-search. Mostly, because the new site-search is so much better, there's no need. 

I can't think of any reason to keep the `tags` in the translated content. But I'm only 95% certain and it would take some time and investment to make that 100%. 

Perhaps you can prove me wrong. Perhaps there's some `.ejs` macro that actually does depend on the *translated* document's tags (unlike, say, the CSSRef sidebar which always relies on the tags in the `en-US` docs)